packages/api/src/schema/api-schema.yaml

```
@@ -3466,6 +3464,26 @@ paths:
               $ref: "#/components/schemas/new-asset-from-url-payload"
       responses:
         "200":
+          description: Import in progress
```

Let's call it `Upload` for consistency
```suggestion
          description: Upload in progress
```